### PR TITLE
kubernetes: change OOM test parameter to ensure it will be success

### PR DESCRIPTION
**Pull request by liubin**
Let container run more longer, and if fails, get the pod full yaml spec.

Fixes: #3348

Signed-off-by: bin liub <bin@hyper.sh>

### CODE REVIEW

**Review by Tim-Zhang — APPROVED** [review]
lgtm

**Review by lifupan — APPROVED** [review]

**Review by bergwolf — APPROVED** [review]

**Comment by egernst** [comment]
Still seeing the test fail on Ubuntu:  http://jenkins.katacontainers.io/blue/organizations/jenkins/kata-containers-2.0-tests-ubuntu-PR/detail/kata-containers-2.0-tests-ubuntu-PR/633/pipeline#log-1737

File: integration/kubernetes/runtimeclass_workloads/pod-oom.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       imagePullPolicy: Always
       name: oom-test
       command: ["/bin/sh"]
-      args: ["-c", "sleep 2; stress --vm 2 --vm-bytes 400M --timeout 5s"]
+      args: ["-c", "sleep 2; stress --vm 2 --vm-bytes 400M --timeout 60s"]
```
Comment (egernst) [inline integration/kubernetes/runtimeclass_workloads/pod-oom.yaml:20]: Can we just simplify this and set the memory limit to 250Mi, and run a single instance of --vm?

Comment (egernst) [reply integration/kubernetes/runtimeclass_workloads/pod-oom.yaml:20]: I see this take a long time to reach OOMKill, as the second stress thread takes a while to ramp up.  Let's just make sure the application can abuse, and abuse in a short amount of time?

**Comment by egernst** [comment]
As this seems to be blocking a lot of tests -- I pushed an extra commit. Let's see if this results in passing tests!

**Comment by egernst** [comment]
I think there's some odd behavior around OOMKill handling. I see the TaskEvent in the containerd logs, but the error state when running in CI is Terminated with Error, rather than oomkill.

**Comment by liubin** [comment]
Found these errors in job [413](http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-ubuntu-initrd-PR/413/):

```
2021-03-26T20:13:50.720416318+00:00 kata-agent INFO agent signals: notify term to close
2021-03-26T20:13:50.722546811+00:00 kata-agent INFO agent cgroups_notifier: cgroup event for container: 7a0d5a724c0122fc5e6f99ad965a51ac58df0790569179853911ce30d1ecf16a, path: "/sys/fs/cgroup/memory//kubepods/besteffort/podaeb7c27d-ef7b-45b7-a1f9-cd5bea1b2897/7a0d5a724c0122fc5e6f99ad965a51ac58df0790569179853911ce30d1ecf16a/memory.oom_control", content: Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })
2021-03-26T20:13:50.722546811+00:00 kata-agent INFO agent cgroups_notifier: cgroup event for container: 7a0d5a724c0122fc5e6f99ad965a51ac58df0790569179853911ce30d1ecf16a, path: "/sys/fs/cgroup/memory//kubepods/besteffort/podaeb7c27d-ef7b-45b7-a1f9-cd5bea1b2897/7a0d5a724c0122fc5e6f99ad965a51ac58df0790569179853911ce30d1ecf16a/memory.oom_control", content: Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })
2021-03-26T20:13:50.733942570+00:00 kata-agent INFO agent rpc: signal process
2021-03-26T20:13:50.733942570+00:00 kata-agent INFO agent rpc: signal process
2021-03-26T20:13:50.735189806+00:00 kata-agent INFO agent rpc: wait process
2021-03-26T20:13:50.735189806+00:00 kata-agent INFO agent rpc: wait process
2021-03-26T20:13:50.747924163+00:00 kata-agent INFO agent cgroups_notifier: cgroup event for container: fd126e2b5f00dcbc7b0366810d2ce199672b2b16ae53e9d51e48ad236f3b3a3f, path: "/sys/fs/cgroup/memory//kubepods/besteffort/podd83bf4db-149b-44ff-99b1-21dcabc6d587/fd126e2b5f00dcbc7b0366810d2ce199672b2b16ae53e9d51e48ad236f3b3a3f/memory.oom_control", content: Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })
2021-03-26T20:13:50.747924163+00:00 kata-agent INFO agent cgroups_notifier: cgroup event for container: fd126e2b5f00dcbc7b0366810d2ce199672b2b16ae53e9d51e48ad236f3b3a3f, path: "/sys/fs/cgroup/memory//kubepods/besteffort/podd83bf4db-149b-44ff-99b1-21dcabc6d587/fd126e2b5f00dcbc7b0366810d2ce199672b2b16ae53e9d51e48ad236f3b3a3f/memory.oom_control", content: Err(Os { code: 2, kind: NotFound, message: "No such file or directory" })

```

**Comment by liubin** [comment]
Close this and use the new one https://github.com/kata-containers/tests/pull/3356

**Pull request closed without merging**